Directory.Build.props

```diff
@@ -382,6 +381,11 @@
     <DocumentationFile Condition="'$(DocumentationFile)' == ''">$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsSourceProject)' == 'true' OR '$(IsReferenceAssembly)' == 'true'">
+    <Features Condition="'$(Features)' == ''">nullablePublicOnly</Features>
+    <Features Condition="'$(Features)' != ''">$(Features);nullablePublicOnly</Features>
```

Comment:
Why not just have `<Features>$(Features);nullablePublicOnly</Features>`? Is the additional comma causing issues if none features are set?

Reply:
I haven't tested that. Let me try it.

Reply:
It seems like the csc task correctly handles separator with empty values. WIll update.